### PR TITLE
Upgraded Jersey and Jackson jars

### PR DESCRIPTION
**Pull request by Sandy247**
Refer to https://docs.google.com/document/d/11LYWsPVosKgHBIcAmsNn1fuclR9mONp6vswBIKXAtDE/edit?usp=sharing for details

### CODE REVIEW

**Comment by amoghmargoor** [comment]
Please remove files related to ./idea/ They shouldn't be checkedin

File: qds-sdk-java.iml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (amoghmargoor) [inline qds-sdk-java.iml]: remove this file

File: src/test/java/com/qubole/qds/sdk/java/TestClient.java [hunk 2]
```diff
@@ -100,10 +101,10 @@ protected WebTarget prepareTarget(ForPage forPage, RequestDetails entity, String
         client.command().startDate(startDate).endDate(endDate).allUsers(allUsers).includeQueryProperties(qProps).history().invoke();
         WebTarget webTarget = webTargetReference.get();
         Assert.assertNotNull(webTarget);
-        Assert.assertTrue(webTarget.getUri().toString().contains("end_date="+endDate));
+/*       Assert.assertTrue(webTarget.getUri().toString().contains("end_date="+endDate));
```
Comment (amoghmargoor) [inline src/test/java/com/qubole/qds/sdk/java/TestClient.java]: why was this commented ?

File: target/checkstyle-cachefile [hunk 3]
```diff
@@ -0,0 +1,146 @@
+#Thu Nov 15 14:38:41 IST 2018
```
Comment (amoghmargoor) [inline target/checkstyle-cachefile]: please remove this file

**Comment by amoghmargoor** [comment]
please remove everything under target ...that shouldn't be checkedin.

**Comment by amoghmargoor** [comment]
build is failing even for java8 ..that needs to be fixed.

File: pom.xml [hunk 4]
```diff
@@ -76,6 +77,23 @@
             <version>${jersey-version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>${jackson.version}</version>
+        </dependency>
+a
```
Comment (eberls) [inline pom.xml]: Eliminate spurious 'a'.

**Pull request closed without merging**